fk/luae_na/src/test/java/se/inera/intyg/common/luae_na/v1/model/converter/certificate/question/QuestionNyBedomningDiagnosgrundTest.java

```diff
@@ -0,0 +1,176 @@
+
+            @Override
+            protected CertificateDataElement getElement() {
+                return QuestionNyBedomningDiagnosgrund.toCertificate(true, 0, textProvider);
+            }
+
+            @Override
+            protected String getId() {
+                return NYDIAGNOS_SVAR_ID_45;
+            }
+
+            @Override
+            protected String getParent() {
+                return DIAGNOS_CATEGORY_ID;
+            }
+
+            @Override
+            protected int getIndex() {
+                return 0;
+            }
+        }
+
+        @Nested
+        class IncludeConfigRadioBooleanTest extends ConfigRadioBooleanTest {
+
+            @Override
+            protected String getId() {
+                return DIAGNOSGRUND_NY_BEDOMNING_SVAR_JSON_ID_45;
+            }
+
+            @Override
+            protected CertificateTextProvider getTextProviderMock() {
+                return textProvider;
+            }
+
+            @Override
+            protected CertificateDataElement getElement() {
+                return QuestionNyBedomningDiagnosgrund.toCertificate(true, 0, textProvider);
+            }
+
+            @Override
+            protected String getTextId() {
+                return DIAGNOSGRUND_NYBEDOMNING_SVAR_TEXT_ID;
+            }
+
+            @Override
+            protected String getDescriptionId() {
+                return DIAGNOSGRUND_NYBEDOMNING_SVAR_DESCRIPTION_ID;
+            }
+        }
+
+        @Nested
+        class IncludeValueBooleanTest extends ValueBooleanTest {
+
+            @Override
+            protected String getJsonId() {
+                return DIAGNOSGRUND_NY_BEDOMNING_SVAR_JSON_ID_45;
+            }
+
+            @Override
+            protected Boolean getBoolean() {
+                return true;
+            }
+
+            @Override
+            protected CertificateDataElement getElement() {
+                return QuestionNyBedomningDiagnosgrund.toCertificate(true, 0, textProvider);
+            }
+
+        }
+
+        @Nested
+        class IncludeValidationMandatoryTest extends ValidationMandatoryTest {
+
+            @Override
+            protected String getQuestionId() {
+                return DIAGNOSGRUND_NY_BEDOMNING_SVAR_JSON_ID_45;
+            }
+
+            @Override
+            protected String getExpression() {
+                return singleExpression(DIAGNOSGRUND_NY_BEDOMNING_SVAR_JSON_ID_45);
+            }
+
+            @Override
+            protected CertificateDataElement getElement() {
+                return QuestionNyBedomningDiagnosgrund.toCertificate(true, 0, textProvider);
+            }
+
+            @Override
+            protected int getValidationIndex() {
+                return 0;
+            }
+        }
+    }
+
+    @Nested
+    class ToInternal {
+
+        @Nested
+        @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+        class IncludeInternalRadioBooleanTest extends InternalRadioBooleanTest {
```

Comment:
I think a InternalBooleanValueTest has been introduced in another PR which could be used here. There is no need for a variant with RadioBoolean... It's still a boolean.